merino/jobs/navigational_suggestions/utils.py

```diff
@@ -95,3 +97,27 @@ def update_top_picks_with_firefox_favicons(
         firefox_favicon: Optional[str] = FIREFOX_PACKAGED_FAVICONS.get(domain)
         if firefox_favicon:
             domain_metadata["icon"] = firefox_favicon
+
+
+def _blocklist_path() -> str:
+    return os.path.join(
+        os.path.dirname(__file__),
+        "data/domain_blocklist.json",
+    )
+
+
+def load_blocklist(path: Optional[str] = None) -> set[str]:
+    """Load the blocklist into a set."""
+    if path is None:
+        path = _blocklist_path()
```

Comment:
```suggestion
def load_blocklist(path: str = _blocklist_path()) -> set[str]:
    """Load the blocklist into a set."""
```

I think this might work.